gama_fields.py

```diff
@@ -0,0 +1 @@
+gama_fields = ['G9', 'G12', 'G15']
```

Comment:
Good to have this in gama_limits instead, I'd leave fields (already defined there) for the time being to not have everything fall over.